src/032_longest_valid_parentheses.md

```diff
@@ -0,0 +1,62 @@
+# 032.Longest Valid Parentheses
+
+**<font color=red>level: Hard</font>**
+
+## Description
+
+Given a string containing just the characters '(' and ')', find the length of the longest valid (well-formed) parentheses substring.
+
+Example 1:
+
+Input: "(()"   
+Output: 2  
+Explanation: The longest valid parentheses substring is "()"
+
+Example 2:
+
+Input: ")()())"  
```

Comment:
remove whitespace in the end of line  17?

Reply:
I see...It is for a newline.